.eslintrc.js

```diff
@@ -1,25 +1,29 @@
 module.exports = {
   root: true,
-  parserOptions: {
-    parser: "babel-eslint",
-    sourceType: "module",
-  },
   env: {
-    browser: true,
-    node: true,
-  },
-  extends: ["plugin:vue/recommended", "standard", "plugin:cypress/recommended"],
-  globals: {
-    GIT_DATE: true,
-    GIT_VERSION: true,
+    node: true
   },
+  plugins: ["@typescript-eslint", "prettier", "vue"],
+
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:vue/recommended",
+    "@vue/typescript",
+    "@vue/prettier",
+    "prettier/@typescript-eslint"
+  ],
   rules: {
-    // allow paren-less arrow functions
-    "arrow-parens": 0,
-    // allow async-await
-    "generator-star-spacing": 0,
-    // allow debugger during development
-    "no-debugger": process.env.NODE_ENV === "production" ? 2 : 0,
-    "vue/component-name-in-template-casing": ["warn", "PascalCase", {}],
+    "no-console": process.env.NODE_ENV === "production" ? "error" : "off",
```

Comment:
I rather not have those production checks, as that will result in issues when people don't lint it with the right `NODE_ENV`